GOVERNANCE.md

```diff
@@ -11,21 +11,46 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "Organization Voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, we follow these steps.
+
+1. Have enough period to discuss the topic so that all discussion members agree that there are no more points to be discussed.
+2. Add a specific statement of what is being voted on to the relevant GitHub issue or PR.
+3. Declare the start of voting.
+4. Ask maintainers to indicate their yes/no vote on the issue or PR.
```

Comment:
> Most of these texts follow the current ones.

> https://github.com/fluent/fluentd/blob/dc7e4b295270d6a521f43331321628dba2692ca4/GOVERNANCE.md#L22

This too.

```
Maintainers should indicate their yes/no vote on that issue or PR
```

Do you mean we should fix it on this occasion?

I thought it didn't need to be the focus of this PR.

Reply:
Maybe we should decide the voting style later in the dedicated issue. 
In my opinion, we have to write down the guideline to show yes/no voting. 
`:+1:` and `:-1:` signs should be enough in the most cases.

So, my suggestions is:

Maintainers should indicate their yes/no votes with formally defined signs. The signs must be defined when the voting is started. 

---

Or, we should define something like similar sentences.

Reply:
I agree about making such improvements, as @tagomoris and @cosmo0920 say.

I think it should be made separately from this PR since this is a major rule revision.